influence_benchmark/RL/base_iteration.py

```diff
@@ -345,6 +354,68 @@ def _select_and_format_trajectories(
         )
         bottom_n_dict = get_selected_turns_df(turns_df, bottom_n_df).to_dict("records")
         self._format_and_save_trajectories((top_n_dict, bottom_n_dict), trajectory_iteration_dir)
+        self._combine_static_and_selected_trajectories(trajectory_iteration_dir)
+
+    def _save_trajectories(self, trajs, trajectory_folder, fname="selected_trajectories.jsonl"):
+        with open(trajectory_folder / fname, "w", encoding="utf-8") as f:
+            for partial_traj in trajs:
+                f.write(json.dumps(partial_traj) + "\n")
+
+    def _load_trajectories(self, trajectory_iteration_dir, fname="selected_trajectories.jsonl"):
+        trajectory_file = trajectory_iteration_dir / fname
+        return [json.loads(line) for line in trajectory_file.read_text(encoding="utf-8").splitlines()]
+
+    def _combine_static_and_selected_trajectories(
+        self,
+        trajectory_iteration_dir,
+        static_dataset_name="PKU-Alignment/PKU-SafeRLHF",  # "Anthropic/hh-rlhf",
```

Comment:
This should also be a config flag probably. There may be a nice way to make it a single flag together with the amount of mixing (e.g. a tuple?)